### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by mattock**
Signed-off-by: Samuli Seppänen <samuli@openvpn.net>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -53,7 +54,10 @@ There are three ways to do this:
   system.
 * Place the file into *C:\\Users\\username\\OpenVPN\\config\\*, or any of its
   immediated subdirectories. The configuration file is only visible for the
-  user in question.
+  user in question. If the user is not a member of "OpenVPN Administrators"
```
Comment (selvanair) [inline README.rst]: .... member of the built-in "Administrators" group or "OpenVPN Administrators" group ....

File: README.rst [hunk 2]
```diff
@@ -53,7 +54,10 @@ There are three ways to do this:
...
+  group and tries to launch such a connection, OpenVPN GUI pops up a UAC,
+  offering to create the group (if missing) and to add the user to it. This
```
Comment (selvanair) [inline README.rst]: offering to create the latter group (......

File: README.rst (hunk 1 above)
Comment (mattock) [inline README.rst]: Will fix.

**Comment by mattock** [comment]
@selvanair looks good now?

**Pull request merged**